plugins/buildx.installer

```diff
@@ -18,8 +19,10 @@ build() {
         git fetch --all
         git checkout -q "${COMMIT}"
         local LDFLAGS
-        # TODO: unmark `-tp` when no longer a technical preview
-        LDFLAGS="-X ${PKG}/version.Version=$(git describe --match 'v[0-9]*' --always --tags)-tp -X ${PKG}/version.Revision=$(git rev-parse HEAD) -X ${PKG}/version.Package=${PKG}"
+        local VERSION
```

Comment:
Might also be useful to name this something different as to avoid overwriting the global `VERSION` we use for the dockerd/docker-cli builds.